### ISSUE TITLE
I'm not getting a PDF summary of the results

### ISSUE THREAD

**Issue by PeacefulCoder01** [broadinstitute/CellBender#86]
Hello, I tried to use the tool on several different samples (with the mtx files).
Sometimes it works and I get the output result except the PDF summary, it prints:
"Unable to save plot."
I checked where it fell and I saw it happened in function SingleCellRNACountsDataset.save_to_output_file, line 735:
                z_pca = pca.fit_transform(z[p >= 0.5])

I printed the exception message and it was: 
"array must not contain infs or NaNs"

Would appreciate if you could advice what I am doing wrong? I tried with --cuda and without cuda and it's the same...
Thank you!!

**Comment by sjfleming** [comment]
Hi @itayShlos , this is very strange.  I haven't seen it before!  I wonder: do you get a NaN error during training?  Or does the training process complete as expected, but then you just don't get the PDF due to this exception?  Thanks for tracking down the exception, that's very helpful.  (It looks like there must be an `inf` or `NaN` value in `z`, which is something I have not seen before, and really should not happen.)

**Comment by itayShlos** [comment]
Thanks for the help :)
The training process completed as expected. But actually, when I ran CellBender on another server it seemed to work.  so, it no longer bothers me.

Though, other issue I've had is that CellBender seems to cut a lot of the data (determining droplets as empty) much more than expected. I exemined it with ~40 samples, and the portion of empty droplets quite high. it ranges from ~35% to 91% of the droplets.
Does this behaviour make sense?
